src/data-types.js

```diff
@@ -87,6 +87,18 @@ export function getTypes(options) {
 		}
 	}
 
+	var int64 = {
+		unpack: function(buffer){
+			var value = buffer.readUInt64LE(buffer.pointer);
```

Comment:
Does this code even run? Node does NOT have a `buffer.readUInt64LE()` function! https://nodejs.org/api/buffer.html